pennylane/devices/default_qubit.py

```diff
@@ -620,17 +647,25 @@ def _apply_state_vector(self, state, device_wires):
         """Initialize the internal state vector in a specified state.
 
         Args:
-            state (array[complex]): normalized input state of length
-                ``2**len(wires)``
-            device_wires (Wires): wires that get initialized in the state
+            state (array[array[complex]]): normalized input states
+            device_wires (array[Wires]): wires that get initialized in the state
         """
-
+        states = state
+        if type(device_wires) != list:
```

Comment:
Two points here:
1. Would we allow lists here or arrays? The docstring mentions the latter. If we allow multiple options (e.g., even tuples), then we could potentially check for `Sequence` here;
2. It might be better to check with `isinstance`, see [here for some details](https://stackoverflow.com/a/1549854/12899253).